ReceiptRepository/Tests/PrepaidCardReceiptRepositoryTests.swift

```diff
@@ -0,0 +1,88 @@
+//
+// Copyright 2018 - Present Hyperwallet
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software
+// and associated documentation files (the "Software"), to deal in the Software without restriction,
+// including without limitation the rights to use, copy, modify, merge, publish, distribute,
+// sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all copies or
+// substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+// BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+import Hippolyte
+import HyperwalletSDK
+@testable import ReceiptRepository
+import XCTest
+
+class PrepaidCardReceiptRepositoryTests: XCTestCase {
+    private lazy var listPrepaidCardReceiptPayload = HyperwalletTestHelper.getDataFromJson("PrepaidCardReceiptResponse")
+    private var factory: ReceiptRepositoryFactory!
+    private var prepaidCardReceiptRepository: PrepaidCardReceiptRepository!
+    private var receiptExpectation: XCTestExpectation!
+
+    override func setUp() {
+        Hyperwallet.setup(HyperwalletTestHelper.authenticationProvider)
+        factory = ReceiptRepositoryFactory.shared
+        prepaidCardReceiptRepository = factory.prepaidCardReceiptRepository()
+        receiptExpectation = self.expectation(description: "load prepaid card receipts")
+    }
+
+    override func tearDown() {
+        ReceiptRepositoryFactory.clearInstance()
+        if Hippolyte.shared.isStarted {
+            Hippolyte.shared.stop()
+        }
+    }
+
+    func testListUserReceiptSuccess() {
```

Comment:
rename to testListPrepaidCardReceipt_success

Reply:
renamed